lib/screens/send/send_to.dart

```diff
@@ -100,6 +117,16 @@ class _SendToScreenState extends State<SendToScreen> {
     walletLogic.updateAddress();
```

Comment:
handle if there is a scan result. This screen already calls a scanner to parse, so you can probably just pass the scan result directly